charts/kube-prometheus-stack/templates/alertmanager/servicemonitor.yaml

```
@@ -8,6 +8,7 @@ metadata:
     app: {{ template "kube-prometheus-stack.name" . }}-alertmanager
 {{ include "kube-prometheus-stack.labels" . | indent 4 }}
 spec:
+  {{- include "servicemonitor.scrapeLimits" .Values.alertmanager.serviceMonitor | nindent 2 }}
```

I would prefer to have scrapelimits combined in a "scrapeLimits" var in every servicemonitor.

```suggestion
  {{- include "servicemonitor.scrapeLimits" .Values.alertmanager.serviceMonitor.scrapeLimits | nindent 2 }}
```